CONTRIBUTING.md

```diff
@@ -1,5 +1,13 @@
 # Contributing
 
+<div style="display: inline-block; padding: 4px 8px; background-color: yellow; font-size: 1.5rem; margin-bottom: 1rem; border-radius: 8px">
+  <strong>NOTE</strong>
+  <ul>
+    <li style="margin: 0"><strong>master</strong> branch will be frozen and no further releases will go out on that branch.</li>
```

Comment:
Could we reword the text to the following:
The 'master' branch is being locked, and there won't be any more releases from that branch. Kindly initiate new pull requests against the 'next' branch instead.

Reply:
Updated the text